CONTRIBUTING.md

```diff
@@ -2,35 +2,61 @@
 
 We are really glad you are reading this, because we need volunteer developers to help this project come to fruition.
 
-- [Code of Conduct](#code-of-conduct)
-- [Community](#community)
-- [Issues](#issues)
-  - [Triaging](#triaging)
-    - [More about labels](#more-about-labels)
-- [Pull Requests](#pull-requests)
-  - [Commit convention](#commit-convention)
-    - [Rule type](#rule-type)
-- [Coding Guidelines](#coding-guidelines)
-  - [C++](#c)
-- [Developer Certificate Of Origin](#developer-certificate-of-origin)
+- [How to contribute](#how-to-contribute)
+  - [Code of Conduct](#code-of-conduct)
+  - [Community](#community)
+  - [Contributing](#contributing)
+    - [Contributor of the month](#contributor-of-the-month)
+    - [Issues](#issues)
+    - [Triaging](#triaging)
+      - [More about labels](#more-about-labels)
+    - [Pull Requests](#pull-requests)
+    - [Reviews](#reviews)
+    - [Commit convention](#commit-convention)
+      - [Rule type](#rule-type)
+  - [Coding Guidelines](#coding-guidelines)
+  - [Developer Certificate Of Origin](#developer-certificate-of-origin)
+  - [How do I become a maintainer?](#how-do-i-become-a-maintainer)
 
 ## Code of Conduct
 
 Falco projects have a
 [Code of Conduct](CODE_OF_CONDUCT.md)
 to which all contributors must adhere, please read it before interacting with the repository or the community in any way.
 
-### Community
+## Community
 
 Discussion and **support requests** should go through the `#falco` channel in the [Kubernetes slack](https://slack.k8s.io).
 
 [Join the **mailing list**](https://lists.cncf.io/g/cncf-falco-dev/) for news, announcements, and a Google calendar invite for our Falco open source meetings.
 
-We also run bi-weekly **community calls** which are open calls to discuss Falco projects from an user perspective. In case you want to discuss a topic during those calls you can simply propose it opening an issue in the [community](https://github.com/falcosecurity/community) repository and join the call!
+We also run weekly **community calls** which are open calls to discuss Falco projects from an user perspective. In case you want to discuss a topic during those calls you can simply add it to
+this week's [agenda](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg).
 
-[Wednesdays at 8am Pacific](https://lists.cncf.io/g/cncf-falco-dev/calendar) on [Zoom](https://zoom.us/my/cncffalcoproject).
+[Wednesdays at 4PM UTC](https://lists.cncf.io/g/cncf-falco-dev/calendar) on [Zoom](https://zoom.us/my/cncffalcoproject).
 
-## Issues
+You can subscribe to the calendar with GNOME Calendar, iCal, Google Calendar, or your favorite application using this ICS feed:
+
+```console
+https://lists.cncf.io/g/cncf-falco-dev/ics/4188901/1750099167/feed.ics
+```
```

Comment:
It seems to be redundant with https://github.com/falcosecurity/community#falco-community-calls

Would we remove redundant info from https://github.com/falcosecurity/community#falco-community-calls ?